zproject/computed_settings.py

```diff
@@ -75,7 +76,7 @@
 SECRET_KEY = get_secret("secret_key")
 
 # A shared secret, used to authenticate different parts of the app to each other.
-SHARED_SECRET = get_secret("shared_secret")
+SHARED_SECRET = get_mandatory_secret("shared_secret")
```

Comment:
I think `SECRET_KEY` and `AVATAR_SALT` should also have this adjustment, perhaps in additional commits in this PR.

We can also remove `assert settings.AVATAR_SALT is not None` type lines in those commits.

Reply:
Fixed. I also found some leftover assertions for `SHARED_SECRET`, which are also removed now.